automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath 
```

Comment:
Thanks for the input. These are good suggestions. I'll send updates tomorrow 💯 💯 💯